lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -22,6 +22,7 @@
 #include "glow/Graph/Nodes.h"
 #include "glow/IR/IRUtils.h"
 #include "glow/IR/Instrs.h"
+#include "glow/Quantization/Base/Base.h"
```

Comment:
you need to modify the CMake file accordingly since you introduce a dependency.

Reply:
done. Thanks a lot for your reviews!